docs/narration.rst

```diff
@@ -0,0 +1,157 @@
+=========
+Narration
+=========
+
+Telling the story
+of your actors' screenplay
+falls to the Narrator.
+The Narrator is responsible
+for describing the scene,
+marking the tests,
+and attaching files.
+
+You can fit the Narrator's microphone with adapters
+to send the thrilling account
+to different reporting tools.
+The default Adapter
+provided with ScreenPy
+logs to ``stdout``.
+There may be more Adapters available
+in the :ref:`official extensions` section.
+
+Describing Your Tests and Tasks
+===============================
+
+ScreenPy uses decorators
+to categorize tests
+and narrate actions.
+
+
+``act`` and ``scene``
+---------------------
+
+These two decorators
+can be used to categorize tests.
+Use them in any way
+that makes sense for your team,
+but :ref:`act <Act>`
+is usually meant
+for designating suite-level organization,
+while :ref:`scene <Scene>`
+is meant for smaller groupings.
+
+``beat``
+--------
+
+A :ref:`beat` marks and describes
+an Action your Actor is performing
+or a Question your Actor is asking.
+In other words,
+it marks a test step.
+
+Performables and Answerables
+might reference other
+Performables and Answerables
+in their execution.
+
+``aside``
+---------
+
+:ref:`Asides <aside>` speak a line
+directly to the adapters.
+Use these to add a quick comment
+to the report.
+
+Built-In StdOutAdapter
+======================
+
+The default :ref:`StdOutAdapter`
+logs its narration
+to ``stdout``.
+You can see the full output
+by following the instructions
+for the `ReadTheDocs example <https://github.com/ScreenPyHQ/screenpy_examples/tree/trunk/screenpy/readthedocs>`_
+in the `ScreenPy Examples <https://github.com/ScreenPyHQ/screenpy_examples>`_ collection,
+but here's just the output
+from the ``test_dramatic_moment`` test
+in the :ref:`Complete Example`::
+
+    features/test_mood.py::test_dramatic_moment
+    ----- live log setup -----
+    INFO     screenpy:stdout_adapter.py:42 Cameron arrives on stage!
+    INFO     screenpy:stdout_adapter.py:42 Polly gets over their stagefright!
+    ----- live log call -----
+    INFO     screenpy:stdout_adapter.py:42 Cameron starts recording on Will.
+    INFO     screenpy:stdout_adapter.py:42 Cameron skips to scene #35.
+    INFO     screenpy:stdout_adapter.py:42 Cameron dollies the active camera left.
+    INFO     screenpy:stdout_adapter.py:42 Cameron cuts to a closeup on Will!
+    INFO     screenpy:stdout_adapter.py:42     Cameron zooms in.
+    INFO     screenpy:stdout_adapter.py:42     Cameron jumps to the camera on Will!
+    INFO     screenpy:stdout_adapter.py:42 Cameron executes a dramatic dolly zoom!
+    INFO     screenpy:stdout_adapter.py:42     Cameron performs some thrilling camerawork simultaneously!
+    INFO     screenpy:stdout_adapter.py:42         Cameron dollies the active camera backward.
+    INFO     screenpy:stdout_adapter.py:42         Cameron zooms in.
```

Comment:
Im always hesitant to say stuff like this is too abstract but if you want to paste in some example logging instead of just linking to the examples module, you ought to include something with a `Click` or two

Reply:
This is the logging from the new [step-by-step example suite](https://screenpy-docs.readthedocs.io/en/latest/example/all_together.html), i thought this would be a good place to show what the logging would look like for the test the reader was following. In that suite, `ZoomIn` might as well be `Click`. :P

Reply:
Fair enough!